docs/prms/allpairs.rst

```diff
@@ -8,3 +8,16 @@ with respect to a cartesian product.
 
 All Pairs does not take any arguments. Its source code is at https://github.com/Reed-CompBio/all-pairs-shortest-paths
 licensed under MIT.
+
+Dataset Usage
+-------------
+
+All Pairs Shortest Paths uses ``sources``, ``targets``, and edge weights.
+All Pairs Shortest Paths also considers graph directionality.
```

Comment:
The part about considering a graph's directionality is confusing to me. I think it is better to say this algorithm expects an interactome to be fully directed, fully undirected, or can be a mixed-directionality interacome (which includes directed and undirected edges).